src/firewall.py

```diff
@@ -189,7 +189,10 @@ class YunoUPnP:
     def __init__(self, firewall: "YunoFirewall") -> None:
         self.firewall = firewall
         self.description = "Yunohost firewall"
-        self.upnpc: miniupnpc.UPnP | None = None
+        self.upnpc = miniupnpc.UPnP()
+        self.upnpc.localport = self.UPNP_PORT
```

Comment:
I'm not sure why we define a port here, it works with the default one.

Reply:
I think i kept this from the original code.